runtime/parallel/src/lib.rs

```diff
@@ -630,6 +635,233 @@ impl pallet_multisig::Config for Runtime {
     type WeightInfo = ();
 }
 
+parameter_types! {
+    pub const LaunchPeriod: BlockNumber = 28 * 24 * 60 * MINUTES;
```

Comment:
The period needs to reconsidered.

Reply:
I picked this from https://github.com/paritytech/substrate/blob/3f110196163b5ec03bac5ee188d60bedf3ebd91d/bin/node/runtime/src/lib.rs#L560
This is also the same in Polkadot.

@kaichaosun please let me know what value you want me to put here

Reply:
Let's revisit and change the config later.